gpkit/nomials/math.py

```diff
@@ -343,17 +315,15 @@ def __eq__(self, other):
                 return False
         return super(Monomial, self).__eq__(other)
 
-    # Monomial.__le__ falls back on Posynomial.__le__
```

Comment:
same with these inline comments, why delete?

Reply:
(added back in)